mlflow/server/js/package.json

```diff
@@ -81,13 +83,10 @@
       "lcov"
     ]
   },
-  "proxy": {
-    "/ajax-api": {
-      "target": "http://localhost:5000"
-    },
-    "/get-artifact": {
-      "target": "http://localhost:5000",
-      "ws": true
-    }
-  }
+  "browserslist": [
```

Comment:
Added by browserslist, which is a transitive dependency of react-scripts 2.0 ([see docs](https://facebook.github.io/create-react-app/docs/post-processing-css)).